test/test_prototype_transforms_consistency.py

```diff
@@ -1089,12 +1089,18 @@ def make_datapoints(self, with_mask=True):
         "t_ref, t, data_kwargs",
         [
             (det_transforms.RandomHorizontalFlip(p=1.0), prototype_transforms.RandomHorizontalFlip(p=1.0), {}),
-            (det_transforms.RandomIoUCrop(), prototype_transforms.RandomIoUCrop(), {"with_mask": False}),
+            # FIXME: make
```

Comment:
@pmeier maybe we should not blindly merge between commits. This comment went away between my latest and your latest non-merge commit. What's broken here ?

Reply:
After removing the `Label` and `OneHotLabel` classes, `RandomIoUCrop` no longer removes labels since it doesn't have a notion of them. To resolve this, we need to follow my suggestion in the comment and use a composed transform with `RandomIoUCrop` and the new `SanitizeBoundingBoxes` that @NicolasHug implemented.